pkg/ingress/apisix/tls_test.go

```diff
@@ -69,46 +69,6 @@ spec:
 	assert.EqualValues(t, sslExpect.Group, ssl.Group, "group convert error")
 }
 
-func TestConvert_group_annotation(t *testing.T) {
```

Comment:
Why remove this test case

Reply:
The `ingress.class` now doesn't have the mean for multiple cluster.